tests/monotouch-test/monotouch-test.csproj

```diff
@@ -201,6 +201,7 @@
     <Compile Include="..\common\ConditionalCompilation.cs">
       <Link>ConditionalCompilation.cs</Link>
     </Compile>
+    <Compile Include="AVFoundation\FoureCCTest.cs" />
```

Comment:
I think it was covered above with the 
`<Compile Include="**\*.cs" Exclude="obj\**">`

Reply:
Good point. It was auto added by VSfM and I didn't see that when reviewing.